api.js

```diff
@@ -0,0 +1,136 @@
+const fetch = require('node-fetch');
+
+const WebhookService = require('../services/webhook-service');
+const TranslationService = require('../services/translation-service');
+const { onshapeApiUrl } = require('../utils');
+const redisClient = require('../redis-client');
+    
+const apiRouter = require('express').Router();
+
+/**
+ * Get the Elements of the current document/workspace.
+ * 
+ * GET /api/elements
+ *      -> 200, [ ...elements ]
+ *      -or-
+ *      -> 500, { error: '...' }
+ */
+apiRouter.get('/elements', (req, res) => {
+    fetch(`${onshapeApiUrl}/documents/d/${req.query.documentId}/w/${req.query.workspaceId}/elements`, {
```

Comment:
More of an opinion thing here - this is a perfect opportunity to make this more readable with async/await. ie:
```js
apiRouter.get('/elements', async (req, res) => {
    try {
        const data = await fetch(`${onshapeApiUrl}/documents/d/${req.query.documentId}/w/${req.query.workspaceId}/elements`;
        res.status(200).send(data)
    } catch (err) {
        res.status(500).json({ error: err })
    }
}
```

Reply:
Updated.